frontend/lib/src/components/widgets/AudioInput/AudioInput.tsx

```diff
@@ -206,6 +230,17 @@ const AudioInput: React.FC<Props> = ({
     ]
   )
 
+  useEffect(() => {
+    if (isNullOrUndefined(widgetFormId)) return
+
+    const formClearHelper = new FormClearHelper()
+    formClearHelper.manageFormClearListener(widgetMgr, widgetFormId, () => {
+      handleClear({ updateWidgetManager: true })
```

Comment:
should the form clearing trigger a re-run?

Reply:
Hmm, it most likely shouldn't trigger a rerun since the rerun is already triggered via the submit action of the user. However, we need to update the widget manager with the value. The more correct way would probably be to use `fromUi=false`, but since its in a form it doesn't matter.